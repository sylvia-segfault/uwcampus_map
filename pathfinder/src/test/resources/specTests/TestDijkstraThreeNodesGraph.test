# Find a path with Dijkstra's algorithm on
# a graph that contains three nodes

CreateGraph graph
AddNode graph A
AddNode graph B
AddNode graph C
AddEdge graph A B 6.0
AddEdge graph A B 5.0
AddEdge graph A C 2.0
AddEdge graph A C 3.0
FindPath graph A B
FindPath graph A C