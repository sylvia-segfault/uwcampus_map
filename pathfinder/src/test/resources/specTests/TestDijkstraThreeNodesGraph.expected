# Find a path with Dijkstra's algorithm on
# a graph that contains three nodes

created graph graph
added node A to graph
added node B to graph
added node C to graph
added edge 6.000 from A to B in graph
added edge 5.000 from A to B in graph
added edge 2.000 from A to C in graph
added edge 3.000 from A to C in graph
path from A to B:
A to B with weight 5.000
total cost: 5.000
path from A to C:
A to C with weight 2.000
total cost: 2.000